target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -289,6 +289,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch1" \
 			"0:lan" "1:lan" "2:lan" "3:lan" "4:wan" "7t@eth0"
 		;;
+	mikrotik,rbm11g)
```

Comment:
A switch config for a single port device doesn't make that much sense. Please move the board to the block around line 40, which disables the switch and sets eth0 as lan interface.

Reply:
Makes sense. Didn't notice the devices with disabled switch, thanks.